rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
@yifan-gu Just bumping the comment above to make sure you see it ^^^^

Quotes or no quotes?